docs/intermediate/priming.md

```diff
@@ -0,0 +1,44 @@
+---
+sidebar_position: 8
+---
+
+# 🟢 Priming and Macros
+When interacting with an LLM (such as ChatGPT, which we’ll use as the baseline example for this article), the content of your first input can shape the rest of the conversation. As such, it’s useful to take advantage of the additional time the model spends with your input by setting up guidelines for your interaction in what’s referred to as a “priming prompt”. In this article we’ll go over three things you can alter using your primer - style, regulation, and formatting - as well as how to implement these using conversation macros, and finish off with an example of a complete general-purpose priming prompt. 
```

Comment:
Might be nice to have some example image after first paragraph. Not sure what this should be.